peachjam/templates/peachjam/_ratifications.html

```diff
@@ -0,0 +1,36 @@
+<div>
+
+<div>
+  {% if ratification.source_url %}
+    <p>
+      <strong>Source URL:</strong> <a href="{{ ratification.source_url }}">{{ ratification.source_url }}</a>
+    </p>
+  {% endif %}
+  {% if ratification.last_updated %}
+    <p>
+      <strong>Last updated:</strong> {{ ratification.last_updated.isoformat }}
+    </p>
+  {% endif %}
+</div>
+
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Country</th>
```

Comment:
Translations required throughout this file

Reply:
Added the translations